tests/dialects/test_stencil.py

```diff
@@ -459,35 +447,26 @@ def test_stencil_load_bounds():
 @pytest.mark.parametrize(
     "attr, dims",
     (
-        (
-            i32,
-            ArrayAttr(
-                [IntegerAttr[IntegerType](1, 64), IntegerAttr[IntegerType](2, 64)]
-            ),
-        ),
+        (i32, [64, 64]),
         (
             i64,
-            ArrayAttr(
-                [
-                    IntegerAttr[IntegerType](1, 32),
-                    IntegerAttr[IntegerType](2, 32),
-                    IntegerAttr[IntegerType](3, 32),
-                ]
-            ),
+            [
+                32,
+                32,
+                32,
```

Comment:
nit: remove trailing comma

Reply:
I think this is an automatic black thing

Reply:
I think black only does it when the line gets too long, and not for smaller lines...

Reply:
Indeed, black allows for no trailing comma if it's short enough. good catch, thanks!